OracleUnifiedDirectorySM/README.md

```diff
@@ -0,0 +1,198 @@
+
+## 2.1 Hardware Requirements
+
+| Hardware  | Size  |
+| :-------: | :---: |
+| RAM       | 16GB  |
+| Disk Space| 200GB+|
+
+## 2.2 Software Requirements
+
+|       | Version                        | Command to verify version |
+| :---: | :----------------------------: | :-----------------------: |
+| OS    | Oracle Linux 7.3 or higher     | more /etc/oracle-release  |
+| Docker| Docker version 18.03 or higher | docker version            |
+
+# 3. Prerequisites
+
+## 3.1 Pulling the Oracle JDK (Server JRE) base image
+You can pull the Oracle Server JRE 8 image from the [Oracle Container Registry](https://container-registry.oracle.com). When pulling the Server JRE 8 image, re-tag the image so that it works with the dependent Dockerfiles which refer to the JRE 8 image through oracle/serverjre:8.
+
+**IMPORTANT**: Before you pull the image from the registry, please make sure to log-in through your browser with your SSO credentials and ACCEPT "Terms and Restrictions".
+
+1. Sign in to [Oracle Container Registry](https://container-registry.oracle.com). Click the **Sign in** link which is on the top-right of the Web page.
+2. Click **Java** and then click on **serverjre**.
+3. Click **Accept** to accept the license agreement.
+4. Use following commands to pull Oracle Fusion Middleware infrastructure base image from repository :
+
+        
+        $ docker login container-registry.oracle.com
+        $ docker pull container-registry.oracle.com/java/serverjre:8
+        $ docker tag container-registry.oracle.com/java/serverjre:8 oracle/serverjre:8
+
+## 3.2 Pulling Oracle FMW Infrastructure 12.2.1.4.x image
+You can pull Oracle FMW Infrastructure 12.2.1.4.x image from the [Oracle Container Registry](https://container-registry.oracle.com). When pulling the FMW Infrastructure 12.2.1.4.x image, re-tag the image so that it works with the dependent Dockerfiles which refer to the FMW Infrastructure 12.2.1.4.x image through oracle/fmw-infrastructure:12.2.1.4.0.
+
+**IMPORTANT**: Before you pull the image from the registry, please make sure to log-in through your browser with your SSO credentials and ACCEPT "Terms and Restrictions". fmw-infrastructure images can be found under Middleware section.
+
+1. Sign in to [Oracle Container Registry](https://container-registry.oracle.com). Click the **Sign in** link which is on the top-right of the Web page.
+2. Click **Middleware** and then click on **fmw-infrastructure**.
+3. Click **Accept** to accept the license agreement.
+4. Use following commands to pull Oracle Fusion Middleware infrastructure base image from repository :
+
+        
+        $ docker login container-registry.oracle.com
+        $ docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4-191222
+        $ docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4-191222 oracle/fmw-infrastructure:12.2.1.4.0
+
+# 4. Loading or Building OUDSM Docker Image
+
+## 4.1 Loading OUDSM Docker Image
```

Comment:
This repo is about building the image from a `Dockerfile` not loading it. I'm not sure why this is here.

Reply:
Updated README.md